packages/components/src/css/component/CommandPalette.module.scss

```
@@ -0,0 +1,362 @@
+    transition: background 150ms, color 150ms;
+
+    @include mixin.hover {
+        background: var(--gray-100);
+        color: var(--foreground);
+    }
+}
+
+.commandPaletteTabActive {
+    composes: commandPaletteTab;
+
+    background: var(--primary-100);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)